.ci/release-package-registry-distribution.groovy

```diff
@@ -11,7 +11,6 @@ pipeline {
     DOCKER_REGISTRY_SECRET = 'secret/observability-team/ci/docker-registry/prod'
     DOCKER_TAG = "${params.DOCKER_TAG}"
     DOCKER_IMG_SOURCE = "${env.DOCKER_REGISTRY}/package-registry/distribution:production"
```

Comment:
`DOCKER_IMG_SOURCE` is also only used in the publishing step, would it make sense to move this env variable also there?